src/test/java/junitTest/JunitTest.java

```diff
@@ -0,0 +1,51 @@
+package junitTest;
+
+import static constants.Constants.BOOK_PRODUCT_URL;
+import static constants.Constants.BOOK_TITLE;
+import static constants.Constants.HOME_PAGE_TITLE;
+import static constants.Constants.SEARCH_PAGE_TITLE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import desktop.pageFactory.PageFactoryManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+@Execution(ExecutionMode.CONCURRENT)
+public class JunitTest {
```

Comment:
This class can be renamed to JUnitTest, just because it is a proper noun for this lib